include/functions.h

```diff
@@ -1588,7 +1557,7 @@ void func_800FAAB4(GlobalContext* globalCtx, u8 arg1);
 // void func_800FAC20(void);
 // void func_800FAF74(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE1 param_5);
 // void func_800FB010(void);
-void func_800FB320(GlobalContext* globalCtx, u8 param_2);
+void Kankyo_AddLightningBolts(GlobalContext* globalCtx, u8 param_2);
```

Comment:
```suggestion
void Environment_AddLightningBolts(GlobalContext* globalCtx, u8 num);
```
Functions from `Kankyo_` should have the `Environment_` prefix (I know it was there previously and not related to this PR but I'm also glaring at that `Kankyo_Init` I see above)